.env.example

```diff
@@ -1 +1,6 @@
 FONTS_URL=https://...
+SERVER_ADDRESS = "127.0.0.1"
+# SERVER_ADDRESS = "0.0.0.0"
+PANEL_PORT = 5000
+FASTAPI_PORT = 8000
+PROXY_PANEL_THROUGH_FASTAPI = True
```

Comment:
Fehlt hier noch `RENDER_EXTERNAL_HOSTNAME` als Beispiel-Variable?

Reply:
Not really, but I should document where it comes from. And actually check that it should have the same value as `PROXY_PANEL_THROUGH_FASTAPI`